hummingbot/client/ui/interface_utils.py

```diff
@@ -63,16 +62,32 @@ async def start_trade_monitor(trade_monitor):
                     total_trades = len(trades)
                     market_info: Set[Tuple[str, str]] = set((t.market, t.symbol) for t in trades)
                     for market, symbol in market_info:
-                        quote_asset = symbol.split("-")[1]  # Note that the qiote asset of the last pair is assumed to be the quote asset of P&L for simplicity
+                        quote_asset = symbol.split("-")[1]
                         cur_trades = [t for t in trades if t.market == market and t.symbol == symbol]
                         cur_balances = await hb.get_current_balances(market)
                         cur_price = await get_last_price(market.replace("_PaperTrade", ""), symbol)
                         perf = calculate_performance_metrics(symbol, cur_trades, cur_balances, cur_price)
                         return_pcts.append(perf.return_pct)
-                        pnls.append(perf.total_pnl)
+                        # convert to USDT equivalent
+                        usdt_price = None
+                        if quote_asset not in ["USDT", "USDC", "USD"]:
+                            try:
+                                usdt_price = Decimal(str(perf.total_pnl)) * await get_last_price(market.replace("_PaperTrade", ""), f"{quote_asset}-USDT")
```

Comment:
I don't think this is a good approach as USDT trading pair may not always be available, especially for connectors like Balancer. It's probably safer to assume the quote assets of different trading pairs are of the same value (it doesn't make sense in the first place to do strategy like Arb or XEMM on different quote values pairs).

Reply:
okay, I agree. I will revert the change.